spec/models/comment_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  context 'when Comment correctly created' do
+    comment = Comment.new(user: User.new, post: Post.new, content: 'Test')
+    it { expect(comment).to be_valid }
+  end
```

Comment:
You can reuse the definition of `comment` using let in this case, some references as I don't see this in the book:
* https://relishapp.com/rspec/rspec-core/v/2-11/docs/helper-methods/let-and-let
* https://medium.com/@tomkadwill/all-about-rspec-let-a3b642e08d39

Same comment applies to the other files.

```suggestion
RSpec.describe Comment, type: :model do
  let(:comment) { Comment.new(user: User.new, post: Post.new, content: 'Test') }

  context 'when Comment correctly created' do
    it { expect(comment).to be_valid }
  end
```